DuoUniversal/DuoUniversal.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net471</TargetFrameworks>
     <PackageId>DuoUniversal</PackageId>
-    <Version>1.2.5</Version>
+    <Version>1.3.0</Version>
```

Comment:
It is slightly more convenient for me to do the version bump in a dedicated diff, is it a problem to back this change out and handle it within the release flow?

Reply:
no problem at all, I will change. Just wanted to state that we are actually adding a new method with this version bump. But not really important for me